propka/lib.py

```diff
@@ -162,15 +167,11 @@ def loadOptions(*args):
     parser.add_option("-v", "--version", dest="version_label", default="Jan15",
            help="specifying the sub-version of propka [Jan15/Dec19]")
     parser.add_option("-p", "--parameters",dest="parameters", default=pkg_resources.resource_filename(__name__, "propka.cfg"),
-                      help="set the parameter file [%default]")
-    parser.add_option("-z", "--verbose", dest="verbose", action="store_true", default=True,
-           help="sleep during calculations")
-    parser.add_option("-q", "--quiet", dest="verbose", action="store_false",
-           help="sleep during calculations")
-    parser.add_option("-s", "--silent",  dest="verbose", action="store_false",
-           help="not activated yet")
-    parser.add_option("--verbosity",  dest="verbosity", action="store_const",
-           help="level of printout - not activated yet")
+           help="set the parameter file [%default]")
+    parser.add_option("-q", "--no-print", dest="verbosity", action="store_const", const=0, default=1,
```

Comment:
Sorry for late comment, but just noticed this now.   This option was renamed from --quiet to --no-print; but  --quiet seems like a better name, and matches the -q short option.

Also previously -z was first, and -q was second.  In new code -q is first, and -z is second, which makes the history look like there were more changes then there really were.  Would be nice to change ordering of these 2 options to simplify the diff.